deploy/ui/form.json

```
@@ -1724,16 +1724,16 @@
                   "label": "Configure KieExecutorMDB threads",
                   "type": "object",
                   "max": 1,
-                  "jsonPath": "$.spec.objects.servers[*].MDBMaxSession",
+                  "jsonPath": "$.spec.objects.servers[*].KieExecutorMDBMaxSession",
```

Fix here used json path name of property
```suggestion
                  "jsonPath": "$.spec.objects.servers[*].kieExecutorMDBMaxSession",
```

---

Fixed